apps/portlet-quarantine/src/main/resources/locale/portlet/quarantine_fr.properties

```
@@ -0,0 +1 @@
+documents.quarantine.label=Quarantaine
```

```suggestion
documents.dlp.quarantine.label=Quarantaine
```
Put the other labels